stubs/Send2Trash/METADATA.toml

```diff
@@ -1,6 +1,5 @@
 version = "1.8.*"
 upstream_repository = "https://github.com/arsenetar/send2trash"
-partial_stub = true
```

Comment:
This field only changes whether `partial\n` is added to the `py.typed` file by the stub-uploader -- you'll want to tinker with the settings in the `tool.stubtest` table if you want to see what stuff is missing from the stubs